### ISSUE TITLE
Replace the splitONT with seqkit

### ISSUE THREAD

**Issue by baozg** [marbl/verkko#132]
Hi, @skoren

Since the `splitONT` only uses 1 thread, could this be replaced by `seqkit split` for large UL dataset?

**Comment by skoren** [comment]
It could be but I wouldn't expect multiple threads to help much here, I expect it's I/O limited (at least on our system). Does seqtk split complete significantly faster in multi-threaded mode than single-threaded? How long does the currents split take?

**Comment by baozg** [comment]
More than 6 hours. `seqkit split` will split the dataset into `threads` output in parallel, I could do a small benchmark

**Comment by skoren** [comment]
I did some local benchmarking splitting with the current approach vs seqtk splitting all files input by the user at once. Seqtk was slightly faster but then compressing the sequences afterwards made the total runtime longer. It might be faster to run each input file run by the user in parallel but the ONT input has different organizations (sometimes many small files, sometimes a few large ones, sometimes a mix) which would lead to very uneven partitions if we ran in parallel on all the small files without re-grouping them. So I think this is probably not worth the effort to change as it's also not the major bottleneck in verkko.
